marquez_client/client.py

```diff
@@ -318,10 +318,10 @@ def list_tags(self, limit=None, offset=None):
 
     def _mark_job_run_as(self, run_id, action):
```

Comment:
Good catch here! I think here we'll want to restrict the scope to _private_ using double "underscores " `__mark_job_run_as` vs introducing a new enum. To make things easier for the caller, we can add `mark_job_run_as(run_id, run_state)`, similar to how we do it for the java client [`MarquezClient. markRunAs()`](https://github.com/MarquezProject/marquez-java/blob/main/src/main/java/marquez/client/MarquezClient.java#L303)